libraries/AP_RCProtocol/AP_RCProtocol_IBUS.cpp

```diff
@@ -0,0 +1,147 @@
+/*
+  IBUS decoder, based upon SBUS decoder which was based on src/modules/px4iofirmware/sbus.c from PX4Firmware
+  modified for use in AP_HAL_* by Andrew Tridgell, modified for decoding IBUS.
+ */
+/****************************************************************************
+ *
+ *   Copyright (c) 2012-2014 PX4 Development Team. All rights reserved.
```

Comment:
Is it really based on work from PX4? It doesn't look like it. I think you could remove this.

Reply:
Indirectly yes.
I used the code for SBUS from here as a template to make sure things were set up as per the rest of ardupilot.
I didn't want to cause any copyright issues for lack of attribution so I thought it best to leave the copyright text mostly the same.